src/dsp_tools/xmllib/models/resource.py

```diff
@@ -1188,3 +1199,18 @@ def _check_richtext_before_conversion(value: Any, res_id: str, prop_name: str) -
     if not is_string_like(value):
         msg = f"Resource '{res_id}' has a richtext value that is not a string: Value: {value} | Property: {prop_name}"
         warnings.warn(DspToolsUserWarning(msg))
+
+
+def _check_and_fix_collection_input(value: Any, res_id: str, prop_name: str) -> list[Any]:
```

Comment:
I am sure that at some point someone will pass a string into the multiple function. Which will not raise an error but iterate over the string and will confuse everyone for a while until we find the problem.
Therefore I would add this tolerance even if we do not advertise it.